lambda/index.js

```diff
@@ -22,7 +22,7 @@ const LaunchRequestHandler = {
   },
   handle(handlerInput) {
     wasOpened = true;
-    const speakOutput = 'Hier ist deine Einkaufsliste. Was möchstest du tun?';
+    const speakOutput = 'Einkaufsliste, was möchstest du tun?';
```

Comment:
What's the difference between '. Was' and ', was'? Is one quicker than the other?

Reply:
Yes, I noticed a small difference. Alexa takes a longer break after the ending of a sentence than on a comma which I would prefer because you can access the shopping list just a little bit faster.